src/shared/sam/cli/samCliLocalInvoke.ts

```diff
@@ -66,13 +66,13 @@ export class DefaultSamLocalInvokeCommand implements SamLocalInvokeCommand {
         const runDebugger = new Promise<void>((resolve, reject) => {
             return childProcess.start({
                 onStdout: (text: string): void => {
-                    getLogger('debugConsole').info(text)
+                    getLogger('debugConsole').info(text, { raw: true })
                     // If we have a timeout (as we do on debug) refresh the timeout as we receive text
                     params.timeout?.refresh()
                     this.logger.verbose('SAM: pid %d: stdout: %s', childProcess.pid(), removeAnsi(text))
                 },
                 onStderr: (text: string): void => {
-                    getLogger('debugConsole').error(text)
+                    getLogger('debugConsole').error(text, { raw: true })
                     // If we have a timeout (as we do on debug) refresh the timeout as we receive text
                     params.timeout?.refresh()
                     this.logger.verbose('SAM: pid %d: stderr: %s', childProcess.pid(), removeAnsi(text))
```

Comment:
What does the output look like if the logger is set to `verbose` or higher?

Reply:
So verbose won't show anything since we aren't logging those verbose messages to the channel. Setting to "warn" (hides the info messages) shows this:
![Screen Shot 2021-03-18 at 1 55 02 PM](https://user-images.githubusercontent.com/31319484/111696711-dd9f3a00-87f1-11eb-8157-cf30568667bf.png)

Which is in-line with what the JetBrains toolkit shows.
The logs for our integration tests will still look ugly but that's not super important.